requirements.txt

```diff
@@ -1,4 +1,6 @@
-ansible>=2.2
+# ansible 2.2.2.0 inexplicably fails integration tests under tox;
```

Comment:
described at https://github.com/openshift/openshift-ansible/pull/3816#issuecomment-291328973

I really don't want this in here so I'll keep fiddling with it as the CI work comes along.